apps/landingPage/src/components/Footer.js

```diff
@@ -13,23 +13,23 @@ import styled from 'styled-components';
 
 import logo from '../images/logo.png';
 import {
-  termsAndConditions,
-  termsOfUse,
-  privacyPolicy,
-  security,
+  // termsAndConditions,
+  // termsOfUse,
+  // privacyPolicy,
+  // security,
   codeKiwiInstagram,
   codeKiwiTwitter,
   kiwiLinkedIn,
   codeKiwiFacebook,
   codeKiwi,
 } from '../../../../linksConfig';
 
-const links = [
-  { title: 'Terms & Conditions', url: termsAndConditions },
-  { title: 'Terms of Use', url: termsOfUse },
-  { title: 'Privacy Policy', url: privacyPolicy },
-  { title: 'Security', url: security },
-];
+// const links = [
+//   { title: 'Terms & Conditions', url: termsAndConditions },
+//   { title: 'Terms of Use', url: termsOfUse },
+//   { title: 'Privacy Policy', url: privacyPolicy },
+//   { title: 'Security', url: security },
+// ];
```

Comment:
why is this code commented?

Reply:
These links and some other parts, are present in design but still useless placeholders.
And as it was uncertain whether the landing page is shown on presentation or not, I commented it out.

Reply:
Generally it is better to delete this. If you need to recover it, you can get it from the git-log.